tests/services/test_criteria_matcher.py

```
@@ -56,31 +60,32 @@ def _mock_file(filename="test.txt", **stat_kwargs):
 
-
     return _mock_file
 
+
 # ==================================
 # Time-based Criteria Tests
 # ==================================
 
+
 @pytest.mark.parametrize(
     "operator, file_age_minutes, criterion_value_minutes, expected",
```

Ruff (0.14.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)